src/components/TrialBanner.js

```diff
@@ -156,6 +157,11 @@ export const TrialBanner = _.flow(
           button.isExternal ? icon('pop-out', { style: { marginLeft: '0.25rem' } }) : null
         ]),
         div({ style: { marginLeft: '3rem', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
+          (trialState === 'Enrolled') && h(Clickable, {
+            style: { borderBottom: 'none' },
+            tooltip: 'Hide for now',
```

Comment:
Maybe clarify to "Hide banner" or "Hide banner temporarily" to describe the exact effect of the users action?

Reply:
Adding the temporarily makes the tool tip kinda long, so I'll change it to "Hide Banner"